src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.AppService.tests.ps1

```diff
@@ -0,0 +1,50 @@
+Import-Module -Name $PSScriptRoot\..\Arcus.Scripting.AppService -ErrorAction Stop
+
+InModuleScope Arcus.Scripting.AppService {
+    Describe "Arcus Azure App Service integration tests" {
+        BeforeEach {
+            $filePath = "$PSScriptRoot\appsettings.json"
+            [string]$appsettings = Get-Content $filePath
+            $config = ConvertFrom-Json $appsettings
+            
+            $clientSecret = ConvertTo-SecureString $config.Arcus.ServicePrincipal.ClientSecret -AsPlainText -Force
+            $pscredential = New-Object -TypeName System.Management.Automation.PSCredential($config.Arcus.ServicePrincipal.ClientId, $clientSecret)
+            Disable-AzContextAutosave -Scope Process
+            Connect-AzAccount -Credential $pscredential -TenantId $config.Arcus.TenantId -ServicePrincipal
+        }
+        Context "Setting an Azure App Service application setting" {
+            It "Try to set an application setting to unexisting App Service fails" {
+                # Arrange
+                $ResourceGroupName = $config.Arcus.ResourceGroupName
+                $AppServiceName = "unexisting-appservice"
+                $AppServiceSettingName = "somesetting"
+                $AppServiceSettingValue = "somevalue"
+
+                # Act
+                { Set-AzAppServiceSetting -ResourceGroupName $ResourceGroupName -AppServiceName $AppServiceName -AppServiceSettingName $AppServiceSettingName -AppServiceSettingValue $AppServiceSettingValue -ErrorAction Stop } | 
+                    Should -Throw
+            }
+            It "Setting an application setting succeeds" {
```

Comment:
Maybe we should also add a test to set an application setting that already exists? Then, we can maybe more describe this case case, like: "...new setting..."

Reply:
added 2 seperate test cases for create and update